docs/src/developing_zed__building_zed_windows.md

```diff
@@ -25,7 +25,7 @@ git submodule update --init --recursive
   rustup target add wasm32-wasi
   ```
 
-- Install [Visual Studio](https://visualstudio.microsoft.com/downloads/) with optional component `MSVC v*** - VS YYYY C++ x64/x86 build tools`. 
+- Install [Visual Studio](https://visualstudio.microsoft.com/downloads/) with optional component `MSVC v*** - VS YYYY C++ x64/x86 build tools` and install WINDOWS 11 or 10 SDK depending on your system
```

Comment:
“Windows” should not be written in all caps here.

Reply:
Changed